src/actions/change-methods/showVersionsQuickPick.ts

```diff
@@ -0,0 +1,29 @@
+import * as vscode from 'vscode';
+
+import { CANCEL } from '../../constants';
+
+export default function showVersionsQuickPick({ json, selectedPackage, selectedPackageVersion }: { json: any, selectedPackage: string, selectedPackageVersion: string }): Promise<any | never> {
+    // TODO: This could probably use more error handling.
+    var versions:string[]=[];
+    json.versions.slice().forEach(packageVersion => {
```

Comment:
Why the `slice` call here? `json.versions` doesn't seem to be mutated (unless I missed it somewhere) so I don't think cloning it is necessary.

Reply:
I think I inherited the slice() from src\actions\add-methods\showVersionsQuickPick.ts - but you're right, testing doesn't show any reason to keep slice() for the change-version implementation